README.md

```diff
@@ -643,6 +643,27 @@ $ node -e "console.log(require('./index).main)"
 [Function]
 ```
 
+If you want to be able access modules and their functions from JavaScript land,
+(highly useful for use in `<script>` tags in the browser), then you can use
+a recipe like the following. In your index.js that is input the the bundler
+(e.g. Parcel):
+
+```javascript
+// index.js
+
+import * as Foo from  "./index.opt.js";
```

Comment:
..but there we instruct to use something like `require('./output/Main')`, why recommend this instead?